connectivity/check/policy.go

```diff
@@ -129,11 +129,32 @@ var (
 	// ResultDNSOK expects a successful command, only generating DNS traffic.
 	ResultDNSOK = Result{DNSProxy: true}
 
-	// ResultDNSOKRequestDrop expects a failed command, generating DNS traffic and a dropped flow.
-	ResultDNSOKRequestDrop = Result{DNSProxy: true, Drop: true}
+	// ResultDNSOKDropCurlTimeout expects a failed command, generating DNS traffic and a dropped flow.
+	// 28 is the curl exit code for connection OR DNS resolution timeouts.
+	ResultDNSOKDropCurlTimeout = Result{
+		DNSProxy: true,
+		Drop:     true,
+		ExitCode: 28,
```

Comment:
nit: maybe define these common curl exit codes (22 and 28) as top-level constants, as they are used in multiple places.

Reply:
Done :-)

Reply:
No other changes, so marking ready-to-merge if CI agrees.